_rules/table-headers-attribute-refer-to-data-cells-a25f45.md

```diff
@@ -0,0 +1,287 @@
+---
+id: a25f45
+name: Headers attribute of td element refers to cells in the same table element
```

Comment:
The applicability uses "cell" as a definition, rather than `td`. A cell, according to spec, can be either a `td` or a `th`, so the applicability is inconsistent with the rule name and description. In my opinion this rule shouldn't be limited to `td` elements, so I think the applicability is correct, but the name / description aren't, and there are missing test cases for `th` with a `headers` attribute.